docs/source/community/rfc/rfc-4.rst

```diff
@@ -0,0 +1,1013 @@
+library with a little bit of geodetic capability to a full geodetic
+transformation and description environment has highlighted the importance of
+the support data. Users desire the convenience of software doing the right
+thing with the least amount of fuss, and survey organizations wish to deliver
+their models across as wide a software footprint as possible.  To get results
+with the highest precision, a grid file that defines a model that provides
+dimension shifts is often needed. The proj-datumgrid project centralizes grids
+available under an open data license and bundles them in different archives
+split along major geographical regions of the world .
+
+It is assumed that a PROJ user has downloaded and installed grid files that are
+referred to in the PROJ database. These files can be quite large in aggregate,
+and packaging support by major distribution channels is somewhat uneven due to
+their size, sometimes ambiguous licensing story, and difficult-to-track
+versioning and lineage.  It is not always clear to the user, especially to
+those who may not be so familiar with geodetic operations, that the highest
+precision transformation may not always being applied if grid data is not
+available. Users want both convenience and correctness, and management of the
+shift files can be challenging to those who may not be aware of their
+importance to the process.
+
+The computing environment in which PROJ operates is also changing. Because the
+shift data can be so large (currently more than 700 MB of uncompressed data,
+and growing), deployment of high accuracy operations can be limited due to
+deployment size constraints (serverless operations, for example). Changing to a
+delivery format that supports incremental access over a network along with
+convenient access and compression will ease the resource burden the shift files
+present while allowing the project to deliver transformation capability with
+the highest known precision provided by the survey organizations.
+
+Adjustment grids also tend to be provided in many different formats depending
+on the organization and country that produced them. In PROJ, we have over time
+"standardized" on using horizontal shift grids as NTv2 and vertical shift grids
+using GTX. Both have poor general support as dedicated formats, limited
+metadata capabilities, and neither are not necessarily "cloud optimized" for
+incremental access across a network.
+
+Summary of work planned by this RFC
+-------------------------------------------------------------------------------
+
+- Grids will be hosted by one or several Content Delivery Networks (CDN)
+- Users no longer have to manually fetch grid files and place them in PROJ_LIB.
+  Full and accurate capability of the software will no longer require hundreds
+  of megabytes of grid shift files in advance, even if only just a few of them
+  are needed for the transformations done by the user.
```

Comment:
This is a product of the work, not the actual work. I suggest rephrasing this as a description of what you are going to implement. E.g. "Grid loading mechanism will be reworked to be able to download grids or parts of grids from a online repository... " Something like that.

Reply:
rephrases as
```
Grid loading mechanism will be reworked to be able to download grids or parts
  of grids from a online repository. When opted in, users will no longer have to
  manually fetch grid files and place them in PROJ_LIB.
  Full and accurate capability of the software will no longer require hundreds
  of megabytes of grid shift files in advance, even if only just a few of them
  are needed for the transformations done by the user.
```